### PR TITLE
Attempt to get the CLI tests running in CI

### PR DESCRIPTION
**Pull request by firelizzard18**
By changing the block frequency to ~10Hz

### CODE REVIEW

File: cmd/cli/cmd/root_test.go [hunk 1]
```diff
@@ -118,6 +118,7 @@ func NewTestBVNN(t *testing.T, defaultWorkDir string) (int, int) {
 	require.NoError(t, err)                           //
 	cfg.Accumulate.WebsiteEnabled = false             // Disable the website
 	cfg.Instrumentation.Prometheus = false            // Disable prometheus: https://github.com/tendermint/tendermint/issues/7076
+	cfg.Consensus.TimeoutCommit = time.Second / 10    // Increase block frequency
```
Comment (bunfield) [inline cmd/cli/cmd/root_test.go:121]: I had thought the minimum was 1 second.

Comment (firelizzard18) [reply cmd/cli/cmd/root_test.go:121]: I doubt 100ms is reasonable for any production system, but it doesn't break. I ran a quick test and the actual block frequency I get is more like 5Hz.

File: internal/testing/app_client.go [hunk 2]
```diff
@@ -244,7 +251,10 @@ func (c *ABCIApplicationClient) execute(interval time.Duration) {
 		// Collect any queued up sends
 	collect:
 		select {
-		case sub := <-c.txCh:
+		case sub, ok := <-c.txCh:
+			if !ok {
+				return
+			}
```
Comment (firelizzard18) [inline internal/testing/app_client.go:257]: Make sure the loop exists here if `txCh` has been closed

File: internal/testing/app_client.go [hunk 3]
```diff
@@ -211,22 +211,29 @@ func (c *ABCIApplicationClient) execute(interval time.Duration) {
...
+			close(sub.DidCheck)
+			close(sub.DidDeliver)
+			close(sub.DidCommit)
+			sub.Done = true
+		}
+
+		c.txActive = 0
+		c.txPend = map[[32]byte]bool{}
+		c.txCond.Broadcast()
+	}()
```
Comment (firelizzard18) [inline internal/testing/app_client.go:230]: Move the cleanup to a defer so it happens at all exit points

File: internal/testing/app_client.go [hunk 4]
```diff
@@ -211,22 +211,29 @@ func (c *ABCIApplicationClient) execute(interval time.Duration) {
...
+			}
+			close(sub.DidCheck)
+			close(sub.DidDeliver)
+			close(sub.DidCommit)
+			sub.Done = true
+		}
+
+		c.txActive = 0
+		c.txPend = map[[32]byte]bool{}
+		c.txCond.Broadcast()
```
Comment (firelizzard18) [inline internal/testing/app_client.go:229]: Make sure no one gets stuck waiting

File: cmd/cli/cmd/adi_test.go [hunk 5]
```diff
@@ -35,14 +35,14 @@ func testCase2_1(t *testing.T, tc *testCmd) {
 	testCase5_1(t, tc)
 
 	//need to wait a sec to make sure faucet tx settles
-	time.Sleep(10 * time.Second)
+	time.Sleep(2 * time.Second)
```
Comment (PaulSnow) [inline cmd/cli/cmd/adi_test.go:38]: Gosh, this sure looks like this should be a constant rather than a hard coded value.

**Review by PaulSnow — CHANGES_REQUESTED** [review]

**Review by PaulSnow — APPROVED** [review]

**Pull request merged**